crowbar_framework/app/models/service_object.rb

```diff
@@ -1543,9 +1561,10 @@ def run_remote_chef_client(node, command, logfile_name)
 
   private
 
-  def wait_for_chef_clients(node)
-    unless RemoteNode.chef_ready?(node, 1200)
-      STDERR.puts "Waiting for already running chef-clients on #{node} failed"
+  def wait_for_chef_clients(node_name)
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
+    unless RemoteNode.chef_ready?(node_name, 1200)
```

Comment:
Hrm, I just realized we opened a can of worms: `RemoteNode.chef_ready?` is using `puts` internally :/

Not sure how to best handle that now... @bkutil might have a good idea?